src/main/java/me/eccentric_nz/TARDIS/commands/tardis/TARDISVersionCommand.java

```diff
@@ -33,13 +38,27 @@
     }
 
     boolean displayVersion(CommandSender sender) {
-        String version = plugin.getPM().getPlugin("TARDIS").getDescription().getVersion();
+        final PluginManager pm = Bukkit.getServer().getPluginManager();
```

Comment:
you know, after looking at this, I'm gonna assume plugin.getPM() does this same line...

Reply:
yup